docs/csharp/language-reference/builtin-types/void.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "void - C# reference"
+ms.date: 02/11/2020
+f1_keywords: 
+  - "void_CSharpKeyword"
+  - "void"
+helpviewer_keywords: 
+  - "void keyword [C#]"
+ms.assetid: 0d2d8a95-fe20-4fbd-bf5d-c1e54bce71d4
+---
+# void (C# reference)
+
+You use `void` as the return type of a [method](../../programming-guide/classes-and-structs/methods.md) (or a [local function](../../programming-guide/classes-and-structs/local-functions.md)) to specify that the method doesn't return a value.
+
+[!code-csharp[void method](~/samples/csharp/language-reference/builtin-types/VoidType.cs#VoidExample)]
+
+You can also use `void` as a referent type to declare a pointer to an unknown type. For more information, see [Pointer types](../../programming-guide/unsafe-code-pointers/pointer-types.md).
+
+You cannot use `void` as the type of a variable.
+
+The type keyword `void` is an alias for the .NET <xref:System.Void?displayProperty=nameWithType> type.
```

Comment:
I wonder what this adds. Because it's rare that anyone would use `System.Void` (outside of reflection). It's true, but I don't know how helpful it is. I'm tempted to ask to delete this paragraph.

Reply:
@BillWagner I think at least the link to the .NET type should present on the page, but I agree that the paragraph might be too much. So, I've removed the paragraph and added the link to the **See also** section. What do you think?

Reply:
That's perfect.